.github/workflows/locize-pull-published-sync-pr.yml

```
@@ -0,0 +1,53 @@
+name: Locize Version Published Workflow
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)